### ISSUE TITLE
display of information about hidden panes take up as little space as possible

### ISSUE THREAD

**Issue by cneidle** [DCS-LCSR/SignStream3#39]
Hi,  This is probably self-evident and just something that you haven't yet had a chance to implement, but the reason for hiding panes is to maximize the vertical display space for the one(s) that is/are visible.  So, the amount of space needed to indicate that a pane is hidden should be minimized.  Thanks.

Ideally, if it could be moved entirely into the left area, that would be even better, but at least it should be reduced to not more than one line if visible in the main area.

Thanks!

<img width="1337" alt="screen shot 2015-10-04 at 7 04 47 pm" src="https://cloud.githubusercontent.com/assets/13629122/10270800/8161c7c4-6acb-11e5-9ce2-57daff7352cd.png">

**Comment by gregorydimitriadis** [comment]
As a temporary workaround until this bug is fixed completely, resizing the SignStream window vertically will allow a single non-hidden pane to expand to a greater size (assuming there is space on the screen to do so).

The goal is to allow the panes to act as an accordion and each take up as much space as the others leave behind when hidden, and in addition to have each pane visible/clickable on the left side area.

**Comment by blazetravis** [comment]
This is sort of fixed.

At one window size (see first image) the space is as Carol initially described.

![screen shot 2015-10-07 at 5 51 45 pm](https://cloud.githubusercontent.com/assets/13489587/10352272/29739238-6d1c-11e5-98f6-942bd925ba26.png)

But as I adjust the window to slightly larger dimensions (second image), the panes pop into better alignment.

![screen shot 2015-10-07 at 5 52 56 pm](https://cloud.githubusercontent.com/assets/13489587/10352306/456c60a0-6d1c-11e5-86f6-abee586ee89a.png)

**Comment by gregorydimitriadis** [comment]
This will be completed in the next roll-out for Sprint 4.

**Comment by blazetravis** [comment]
The properties of the panes are now as we desired them. So bug fixed. However, when adding an additional segment tier to test this, the "segment tier 2" is displayed ABOVE the primary segment 1. This was not as before, and I figured this would be important to note.

![screen shot 2015-10-27 at 12 39 30 pm](https://cloud.githubusercontent.com/assets/13489587/10765178/e965dec6-7ca7-11e5-9a55-071fc8d569e4.png)

![screen shot 2015-10-27 at 12 39 47 pm](https://cloud.githubusercontent.com/assets/13489587/10765187/ec6afc28-7ca7-11e5-94f7-43c36dd2f9b6.png)
